src/foam/lib/csv/Outputter.java

```diff
@@ -176,7 +176,7 @@ protected void outputFObject(FObject obj) {
 
     while ( i.hasNext() ) {
       PropertyInfo prop = (PropertyInfo) i.next();
-      prop.toCSV(this, prop.f(obj));
+      output(prop.f(obj));
```

Comment:
This change can change the behaviour of anything using this Outputter. Please make it a priority to remove this Outputter ASAP so nobody tries to fix it if/when something breaks. Anything using this outputter needs to be updated to use the new outputter.